### ISSUE TITLE
Crash report alpha-38: ISE in ImageUtils.resizeImageAndWriteToStream

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#5333]
```
Fatal Exception: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=2000, result=-1, data=Intent { dat=content: flg=0x1 }} to activity {org.wordpress.android/org.wordpress.android.ui.posts.EditPostActivity}: java.lang.IllegalStateException: Can't compress a recycled bitmap
       at android.app.ActivityThread.deliverResults(ActivityThread.java:4013)
       at android.app.ActivityThread.handleSendResult(ActivityThread.java:4063)
       at android.app.ActivityThread.access$1400(ActivityThread.java:150)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1448)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:168)
       at android.app.ActivityThread.main(ActivityThread.java:5885)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:819)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:709)
Caused by java.lang.IllegalStateException: Can't compress a recycled bitmap
       at android.graphics.Bitmap.checkRecycled(Bitmap.java:351)
       at android.graphics.Bitmap.compress(Bitmap.java:1018)
       at org.wordpress.android.util.ImageUtils.resizeImageAndWriteToStream(ImageUtils.java:524)
       at org.wordpress.android.util.ImageUtils.createResizedImageWithMaxWidth(ImageUtils.java:579)
       at org.wordpress.android.ui.posts.EditPostActivity.addMediaVisualEditor(EditPostActivity.java:1625)
       at org.wordpress.android.ui.posts.EditPostActivity.addMedia(EditPostActivity.java:1598)
       at org.wordpress.android.ui.posts.EditPostActivity.fetchMedia(EditPostActivity.java:1503)
       at org.wordpress.android.ui.posts.EditPostActivity.onActivityResult(EditPostActivity.java:1691)
       at android.app.Activity.dispatchActivityResult(Activity.java:6467)
       at android.app.ActivityThread.deliverResults(ActivityThread.java:4009)
       at android.app.ActivityThread.handleSendResult(ActivityThread.java:4063)
       at android.app.ActivityThread.access$1400(ActivityThread.java:150)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1448)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:168)
       at android.app.ActivityThread.main(ActivityThread.java:5885)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:819)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:709)
```

**Comment by maxme** [comment]
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/5308/commits/4fa2fddaa6ab592a5637d3bbe433098e9a9facc6 - will ship with alpha-39

**Comment by maxme** [comment]
Reopening, this happens in 6.9

**Comment by daniloercoli** [comment]
You're right, `ImageUtils` referenced in version 6.9 has the `recycle` bug in it.
We're already fixed in `develop`, and that fix is available in `7.0`.

As per chat coversation, we're closing this issue since it's a rare crash (eventually open a 6.9.1 and cherry pick the changes).

**Comment by maxme** [comment]
Cherry picked the fix in `hotfix/6.9.1`
